src/movies.js

```diff
@@ -1,16 +1,121 @@
-// Iteration 1: All directors? - Get the array of all directors.
+
+  
+  // Iteration 1: All directors? - Get the array of all directors.
+
+  function getAllDirectors(directors) {
```

Comment:
Your parameter "directors", this function will receive an array of movies.
Also in your map function, "person" refers to each movie in the array.

Naming is one of the hardest thing in coding, always name your variable or function for what they are.